gradle.properties

```diff
@@ -25,4 +25,4 @@ systemProp.gradle.internal.testdistribution.writeTraceFile=true
 systemProp.gradle.internal.testdistribution.queryResponseTimeout=PT20S
 
 # Default performance baseline
-defaultPerformanceBaselines=8.1-commit-f6b887c2
+defaultPerformanceBaselines=8.2-commit-85ebea1
```

Comment:
@wolfs let's iterate. I guess it's the [version.txt](https://github.com/gradle/gradle/blob/master/version.txt) where this comes from, plus a short-SHA. This should be a rebaseline on the top commit.

Should this even work?

Reply:
Yes, the base version comes from the `version.txt` the commit you want to rebaseline on is pointing to.

Our build then builds the commit distribution by checking out the commit and assigning the corresponding version to it.

Reply:
If the version the commit is pointing to is `8.2`, then yes.

Reply:
It should

https://github.com/gradle/gradle/blob/85ebea10e4e150ce485184adba811ed3eeaa2622/version.txt#L1

Then the question is if this _makes sense_? 
It makes for me, as what we are trying to achieve here is to make master green by pushing the performance issues aside (to be fixed later).

Reply:
I think it makes sense. One question from me is whether the performance regression shows up on `release` as well, and if it does then we should rebaseline there to a commit on `release`.

Reply:
Release seems to be fine: https://builds.gradle.org/buildConfiguration/Gradle_Release_Check_PerformanceTestTestLinux_Trigger?branch=&mode=builds#all-projects